### ISSUE TITLE
Investigation of global on/off sound option - visual look and location

### ISSUE THREAD

**Issue by emily-phet** [phetsims/a11y-research#76]
Email thread involving @jbphet @ariel-phet and @emily-phet

JB: I’m assuming we will now have mute buttons on most and perhaps all screens for all sims once sonification is integrated.  Are you in agreement with this?  If so, we should probably do some design work on how to add this in a consistent way, since the current button is pretty large and we may want to brainstorm some alternatives.

EM: Yes - all with sound will need the yellow sound on/off button. I think locating it next to the reset all button is a good spot, and consistent with what we’ve already done. In the vast majority of sims I don’t think this will be a problem. My recommendation would be to not try to redesign, but only make adjustments as needed from special case sims where setting next to the reset all button is an issue. A different option would be to consider a sound on/off button in the nav bar. That would require design discussions. (Note - regardless of the decisions, publication of the first sims with auditory display cannot be pushed back to wait on this). 

JB: (main question) Would muting sonification on one screen mute it on all screens?  It seems to me that this is what we would want, and I’m proceeding with the implementation with that in mind, but please let me know if you disagree.
EM: I think this is fine, and would also be consistent with adding the sound on/off button to the nav bar (if that is something we can consider). This would help ensure that teachers using the sim (for example) didn’t have to repeatedly turn the sound off for each screen during use if they don’t want sound at all.

AP: I think for the sounds on/off button we would like to potentially make it a bit smaller, but I feel like with our buttons that is easy to do. Placing next to the reset all button seems good.

One issue with having sound on/off mute sound on all screens is we are breaking with our UI/UX conventions. The reset all button only resets that particular screen, it is not a universal control. It seems we may need some kind of indicator in the navbar at the very least (a speaker on/off) indicator that is small and non-interactive for instance. If this is going to be a universal control, in my opinion it should either be somehow located on the navbar (tricky given our space constraints there) or at the very least it needs to be indicated in a universal manner (such as an indicator on the navbar). 

For the immediate issue, presumably for a sim with sonification, all screens will be sonified, so pressing the sound button will enable/disable it on all screens and the user will see the state of it when they switch screens. So I think that is fine in the short term. In the long term, it seems like such a control should be in a more universal location.

**Comment by emily-phet** [comment]
Global sound button implemented.
